test/WebSites/FilesWebSite/Models/User.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using Microsoft.AspNetCore.Http;
+
+namespace FilesWebSite.Models
+{
+    public class User
+    {
+        public string Name { get; set; }
+        public int Age { get; set; }
+        public IFormFile Biography { get; set; }
+
+        public string ReadBiography()
+        {
+            if (Biography != null)
+            {
+                using (var reader = new StreamReader(Biography.OpenReadStream()))
+                {
+                    return reader.ReadToEnd();
```

Comment:
I agree that this is just a test, but if our plan is to show this as a sample to our users, I think this should be `async`